docs/changelog/127877.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 127877
+summary: Check hidden frames in entitlements
+area: Infra/Entitlements
```

Comment:
I think we need to change this to `Infra/Core` or add `Infra/Entitlements` to the changelog schema